ply2txt.py

```diff
@@ -0,0 +1,15 @@
+#from lib.ply_np_converter import ply2np, np2ply
```

Comment:
This files doesn't seem to belong on this PR or at this location here.  Where does this get used?  If we do need this file it shouldn't go at the root level.

Reply:
Please take this file out of this PR.  We can make a new PR that adds this file, but it should go in `tools`, not at the root.